metricflow/test/compare_df.py

```diff
@@ -1,7 +1,30 @@
+import logging
+
 import math
 import pandas as pd
 
 
+logger = logging.getLogger(__name__)
+
+
+def _dataframes_contain_same_data(
+    actual: pd.DataFrame,
+    expected: pd.DataFrame,
+) -> bool:
+    """Compare all elements of the dataframe one by one, assuming both dataframes have the same shape."""
```

Comment:
Per @lebca's comment:

```suggestion
    """Compare all elements of the dataframe one by one, assuming both dataframes have the same shape.
    
    We use math.isclose() to avoid issues with float comparisons
    """
```

Reply:
Updated.